### PR TITLE
chore: rename 'che' to 'Che' in descriptions, etc.

### PR DESCRIPTION
**Pull request by xcoulon**
Signed-off-by: Xavier Coulon <xcoulon@redhat.com>

### CODE REVIEW

File: manifests/0.1.0/toolchain.openshift.dev_cheinstallations.yaml [hunk 1]
```diff
@@ -44,7 +44,7 @@ spec:
           description: CheInstallationSpec defines the desired state of CheInstallation
           properties:
             cheOperatorSpec:
-              description: The configuration required for che operator
+              description: The configuration required for Che operator
```
Comment (MatousJobanek) [inline manifests/0.1.0/toolchain.openshift.dev_cheinstallations.yaml]: @xcoulon Please, do not change anyting in the released manifests - they are already released - we cannot modify anything there.

Comment (MatousJobanek) [reply manifests/0.1.0/toolchain.openshift.dev_cheinstallations.yaml]: reverting in https://github.com/codeready-toolchain/toolchain-operator/pull/56

Comment (alexeykazakov) [reply manifests/0.1.0/toolchain.openshift.dev_cheinstallations.yaml]: Good catch @MatousJobanek

Comment (xcoulon) [reply manifests/0.1.0/toolchain.openshift.dev_cheinstallations.yaml]: ah right, my apologies! 🤦‍♂

**Pull request merged**